.travis.gotest.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+run_tests() {
+    go test -v -race ./...
+}
+
+run_tests
```

Comment:
Wouldn't this mean that v1 tests are run when making changes / PRs for v2? That seems like work we shouldn't want to do?

Reply:
It does, that's a very good point. My thought was that as a first iteration we just run tests across the board and if we find that they're slowing down builds too much, we can then optimise it to run only tests on what has changed.

A big positive I can see with the current scenario is that it would prevent accidental regressions being backported into v1

Reply:
That's a fair point. Optimizing this later when needed is not a bad thought. :+1: